dealership/contracts.py

```diff
@@ -1,12 +1,54 @@
+from .vehicles import Car, Truck, Motorcycle
+
+
 class Contract(object):
-    pass
+    def __init__(self, vehicle, customer):
+        self.vehicle = vehicle
+        self.customer = customer
+
+    def total_value(self):
+        pass
+
+    def monthly_value(self):
+        pass
 
 
 class BuyContract(Contract):
+    INTEREST_RATES = {
```

Comment:
I like your use of dictionaries for `calculating total_value()` in the `BuyContract` class, as well as for calculating `get_lease_multiplier()` in the `LeaseContract` class. It makes your code very flexible, and and changes that need to be made to these rates can simply be done in the dictionary.